testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeHiveCaching.java

```diff
@@ -86,7 +88,7 @@ public void extendEnvironment(Environment.Builder builder)
     @SuppressWarnings("resource")
     private void createPrestoWorker(Environment.Builder builder, int workerNumber)
     {
-        builder.addContainer(createPrestoContainer(dockerFiles, serverPackage, debug, "ghcr.io/trinodb/testing/centos7-oj11:" + imagesVersion, worker(workerNumber))
+        builder.addContainer(createPrestoContainer(dockerFiles, serverPackage, javaHome, debug, "ghcr.io/trinodb/testing/centos7-oj11:" + imagesVersion, worker(workerNumber))
```

Comment:
Is there some historical reason why these don't extend the already existing worker in multinode provided by `createTrinoWorker`? And instead create their own repeating some config files?

No change requested but if there's no reason maybe we can clean it and extract common configs to StandardMultinode#createTrinoWorker.

Reply:
I guess that this is due to the reason that createPrestoWorker is called twice, so to reduce the duplication there